src/uwtools/tests/test_rocoto.py

```diff
@@ -63,158 +52,160 @@ def test_validate_rocoto_xml(vals):
     assert rocoto.validate_rocoto_xml(input_xml=xml) is validity
 
 
-def test__RocotoXML__doctype_entities(instance):
-    assert '<!ENTITY ACCOUNT "myaccount">' in instance._doctype
-    assert '<!ENTITY FOO "test.log">' in instance._doctype
-
+class Test_RocotoXML:
+    """
+    Tests for class uwtools.rocoto._RocotoXML.
+    """
 
-def test__RocotoXML__doctype_entities_none(instance):
-    del instance._config["workflow"]["entities"]
-    assert instance._doctype is None
+    @fixture
+    def instance(self, assets):
+        cfgfile, _ = assets
+        return rocoto._RocotoXML(config_file=cfgfile)
 
+    @fixture
+    def root(self):
+        return rocoto.Element("root")
```

Comment:
^^^ Here are the two relocated fixtures.

Below this line in this module, and despite the huge diff, nothing has actually changed except that 1. The method names are simplified, removing the `_RocotoXML` component, as that is now obvious due to their membership in this class; and 2. Each function signature now includes the mandatory `self` argument.